sdk/component/internal/storer/local/sqlite/sqlite.go

```diff
@@ -211,6 +215,28 @@ func (m *manager) Close(ctx context.Context) error {
 	return nil
 }
 
+// TODO: potentially leverage migrations here but this is simple enough for now for local setup.
```

Comment:
Could you please write up a ticket on Linear with the next steps and link it here? If not relevant, this can just be a comment, not a todo.

Reply:
Done :)